ivy/functional/frontends/numpy/ndarray/ndarray.py

```diff
@@ -230,6 +230,12 @@ def diagonal(self, *, offset=0, axis1=0, axis2=1):
             axis2=axis2,
         )
 
+    def compress(self, value, *, axis=None):
+        return np_frontend.compress(
```

Comment:
This function is isn't yet implemented in the numpy frontend
Need to implement it in the frontend side before implementing it's instance method

Reply:
i removed compress function and kept only angle. correct me if i am wrong.
do i have to wait for ivy to add compress in frontend or do i have to add compress in frontend

Reply:
Only the function needs to be implemented in the numpy frontend.
If it is not in ivy functional API you can use composition of ivy functions to implement behavior of this function

Reply:
Ahhh now I understood that thank you